Dockerfile

```diff
@@ -1,12 +1,12 @@
+ARG base_image=gcr.io/distroless/static
+
 FROM golang:1.20 as builder
 ARG pack_version
 ENV PACK_VERSION=$pack_version
 WORKDIR /app
 COPY . .
 RUN make build
 
-FROM scratch
```

Comment:
We base the lifecycle image on gcr.io/distroless/static, so it might make sense here as well. It's not as small as scratch but it avoids the need for branching logic in the Dockerfile.